operator/mapper/postgresql.go

```diff
@@ -37,6 +41,30 @@ func (PostgreSQLMapper) FromSpec(spec exoscalev1.PostgreSQLParameters, into *oap
 	return nil
 }
 
+// FromSpecToUpdateBody places the given spec into the request body.
+func (PostgreSQLMapper) FromSpecToUpdateBody(spec exoscalev1.PostgreSQLParameters, into *oapi.UpdateDbaasServicePgJSONRequestBody) error {
+	/**
+	NOTE: If you change anything below, also update FromSpecToCreateBody().
```

Comment:
Have you evaluated to use reflection in this case?

Reply:
I considered several different approaches... new types with interfaces, reflection, some structural changes... eventually I concluded that it doesn't reduce the code and only makes things more complicated. So I left it as is, can still be improved later :shrug: